### DISCUSSION TITLE
ESMValTool at EGU24

### DISCUSSION THREAD

**Discussion by axel-lauer** [ESMValGroup/Community#163] [Announcements]
@ESMValGroup/esmvaltool-developmentteam 

Exciting ESMValTool presentations and community meeting at EGU in Vienna next week! If you are there come and meet us. We would be happy to see you.

https://esmvaltool.org/2024-04-12-EGU-24/

**Comment by valeriupredoi** [comment]
not me this time round, but have lots of fun and wow them all with our shtuffs! :beer:
